packages/manager/modules/nutanix/src/dashboard/general-info/redeploy/controller.js

```diff
@@ -69,44 +68,43 @@ export default class NutanixGeneralInfoRedeployCtrl {
     return modelValue;
   }
 
-  static getIpsList(config = {}) {
+  getIpsList() {
+    if (!this.config) {
+      return [];
+    }
     return [
-      config.prismElementVip,
-      config.prismCentral?.vip,
-      ...(config?.prismCentral?.ips && config.prismCentral.ips),
-      ...(config.nodes?.length > 0 &&
-        config.nodes.reduce((acc, item) => {
+      this.config.prismElementVip,
+      this.config.prismCentral?.vip,
+      ...(this.config?.prismCentral?.ips && this.config.prismCentral.ips),
```

Comment:
Isn't it the same thing twice?

Reply:
No, the first `this.config?.prismCentral?.ips` is a check, if the exists, array is returned and then destructured.